app/frontend/packs/further_conditions/add_another.js

```diff
@@ -0,0 +1,24 @@
+import assignIndexBasedValues from './helper'
+import removeFurtherCondition from './remove'
+
+const addFurtherConditionElement = () => {
+  const items = document.getElementsByClassName('app-add-another__item')
+  const latestId = items.length
+  const newNode = buildNewFurtherConditionField(latestId)
+  document.querySelector('.app-add-another__add-button').insertAdjacentElement('beforebegin', newNode)
+  newNode.querySelector('textarea').focus()
```

Comment:
can we limit the add another to 18 conditions (now its infinite)?

Reply:
Ah good spot

Reply:
Fixed and added JS unit test to cover